src/app/components/navbar/navbar/navbar.component.ts

```diff
@@ -1,48 +1,64 @@
+import { MatDialog } from '@angular/material/dialog';
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
-import { PopUpHowToPlayService } from 'src/app/services/pop-up-how-to-play/pop-up-how-to-play.service';
+import { PopUpHowToPlayComponent } from '../../pop-up-how-to-play/pop-up-how-to-play.component';
 
 @Component({
   selector: 'app-navbar',
   templateUrl: './navbar.component.html',
-  styleUrls: ['./navbar.component.scss'],
+  styleUrls: ['./navbar.component.scss']
 })
 export class NavbarComponent implements OnInit {
   private currentRoute = '';
 
   constructor(
     private router: Router,
-    public myService: PopUpHowToPlayService
+    public dialog: MatDialog
   ) {}
 
-  ngOnInit() {
+  public ngOnInit(): void {
     this.currentRoute = this.router.url;
   }
 
   /**
    * Verifica se a página atual é a página de login.
    * @returns True se a página atual é a página de login, caso contrário, False.
    */
-  isLoginPage() {
+  public isLoginPage(): boolean {
     return this.currentRoute === '/login';
   }
   /**
```

Comment:
Inserir linha em branco entre os métodos

Reply:
Tentei inserir as linhas em branco entre o métodos mas o Lint não está permitindo.

Trailing spaces not allowed.eslint[no-trailing-spaces](https://eslint.org/docs/latest/rules/no-trailing-spaces)
Delete `··`eslint[prettier/prettier](https://github.com/prettier/eslint-plugin-prettier#options)